platforms/opensolar/contract.go

```diff
@@ -480,22 +480,32 @@ func Payback(recpIndex int, projIndex int, assetName string, amount string, reci
 		return fmt.Errorf("other investment models are not supported right now, quitting")
 	}
 
-	err = model.MunibondPayback(consts.OpenSolarIssuerDir, escrowPath, recpIndex, amount, recipientSeed, projIndex, assetName, project.InvestorIndices)
+	pct, err := model.MunibondPayback(consts.OpenSolarIssuerDir, escrowPath, recpIndex, amount,
+		recipientSeed, projIndex, assetName, project.InvestorIndices, project.TotalValue)
 	if err != nil {
 		return errors.Wrap(err, "Error while paying back the issuer")
 	}
 
 	// MW: Ownership of asset could shift as payments happen, or flip at the end.
```

Comment:
we're shifting ownership as payments are made in this model. Can be tweaked based on the outcome that the doc produces